kglm/models/entity_disc.py

```diff
@@ -118,7 +118,7 @@ def forward(self,  # pylint: disable=arguments-differ
                 entity_types: Optional[torch.Tensor] = None,
                 entity_ids: Optional[torch.Tensor] = None,
                 mention_lengths: Optional[torch.Tensor] = None,
-                reset: bool = False)-> Dict[str, torch.Tensor]:
+                reset: torch.ByteTensor=None)-> Dict[str, torch.Tensor]:
```

Comment:
@rloganiv can you verify that this was using an incorrect default before? i guessed so, since line 152 has:

```
        if reset is not None:
            self.reset_states(reset)
```

Reply:
Yes, that was from when reset was a bool - good catch!